data/mock_data.rb

```diff
@@ -0,0 +1,75 @@
+require 'Date'
+
+class MockData
+
+  def self.get_a_random_date(random = true)
+    if random
+      date_s = "20#{rand(10..21)}-#{rand(1..12)}-#{rand(1..28)}"
+      return Date.strptime(date_s, '%Y-%m-%d')
+    else
+      return Date.strptime('2020-01-01', '%Y-%m-%d')
+    end
+  end
+
+  def self.get_a_random_price
+    (rand(1..120) + (rand(100) / 100.0))
+  end
+
+  def self.get_mock_merchants(number_of_mocks: 10, random_dates: true)
```

Comment:
Really like that we have a baseline and can turn off the randomness -- that seems incredibly useful for specific/targeted testing. Nice thought!